test/unit/managers/ConfigManager.ts

```diff
@@ -1,17 +1,32 @@
 import '../../support/polyfills/polyfills';
-import test from 'ava';
+import test, { TestContext } from 'ava';
 import { TestEnvironment, HttpHttpsEnvironment } from '../../support/sdk/TestEnvironment';
 import { ConfigIntegrationKind } from '../../../src/models/AppConfig';
 import ConfigManager from '../../../src/managers/ConfigManager';
 import { AppUserConfig, AppConfig, ServerAppConfig } from '../../../src/models/AppConfig';
 import { AppUserConfigCustomLinkOptions } from '../../../src/models/Prompts';
+import Random from '../../support/tester/Random';
+import { InitTestHelper } from '../../support/tester/utils';
+import sinon, { SinonSandbox } from 'sinon';
+import { ConfigHelper } from '../../../src/helpers/ConfigHelper';
+
+const sinonSandbox: SinonSandbox = sinon.sandbox.create();
+const initTestHelper = new InitTestHelper(sinonSandbox);
 
 test.beforeEach(async () => {
   await TestEnvironment.initialize({
     httpOrHttps: HttpHttpsEnvironment.Https
   });
 });
 
+test.afterEach(function (_t: TestContext) {
+  sinonSandbox.restore();
+
+  OneSignal._initCalled = false;
+  OneSignal.__initAlreadyCalled = false;
+  OneSignal._sessionInitAlreadyRunning = false;
```

Comment:
It looks like we are cleaning these 3 in `test/unit/public-sdk-apis/init.ts`. We should put this in a test helper somewhere so this can be shared.